### PR TITLE
Init models

### PR DESCRIPTION
**Pull request by anhlh-3049**
- Init models add relationship
Screenshot from 2022-05-11 08-14-23

### CODE REVIEW

**Comment by anhlh-3049** [comment]
ready

File: db/migrate/20220510090306_create_rooms.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreateRooms < ActiveRecord::Migration[6.1]
+  def change
+    create_table :rooms do |t|
+      t.string :name
+      t.decimal :price
+      t.text :description
+      t.integer :status
```
Comment (ngocvt-0484) [inline db/migrate/20220510090306_create_rooms.rb]: set default cho status

File: db/migrate/20220510090306_create_rooms.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      t.integer :user_id
```
Comment (ngocvt-0484) [inline db/migrate/20220510090306_create_rooms.rb]: dùng t.references

**Comment by anhlh-3049** [comment]
ready

**Pull request merged**